pkg/image/registryclient/client_test.go

```diff
@@ -320,13 +321,14 @@ func TestRetryFailure(t *testing.T) {
 		t.Fatalf("unexpected: %v %v %#v", m, err, r)
 	}
 
-	// retry four times
+	// verify docker known errors
```

Comment:
"verify docker known errors" -> "verify network error handling"?  This is generic HTTP stuff, it's not even specific to registries.

Reply:
this is errors that docker client knows about